notebooks/fraud_credit_cards.py

```
@@ -0,0 +1,122 @@
+# COMMAND ----------
+
+# MAGIC %pip install colorama==0.4.6 catboost==1.2.0 gecs==0.1.1
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ### Import Libraries
+
+# COMMAND ----------
+
+import seaborn as sns
```

Ruff

`seaborn` imported but unused

Remove unused import: `seaborn`

(F401)